src/QueryBuilder/DSL/Aggregation.php

```diff
@@ -428,4 +429,14 @@ public function diversified_sampler(string $name): DiversifiedSampler
     {
         return new DiversifiedSampler($name);
     }
+
+    /**
+     * composite aggregation.
+     *
+     * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html
+     */
+    public function composite(string $name): Composite
```

Comment:
Should also be added to `Elastica\QueryBuilder\Version\Version700::$aggregations`

Reply:
There's this one to fix too ;)

Reply:
done

Reply:
Thanks ;)